cypress/e2e/loginTest.cy.js

```diff
@@ -0,0 +1,124 @@
+describe('Login/logout functionality and authentication', () => {
+    // Checks if a user can successfully log in
+    it('Login functionality', () => {
+      cy.visit('../../index.html');
```

Comment:
```suggestion
      cy.visit('/l');
```

That should work too